.utility/deploy_docs.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+    pip install --user virtualenv
+    wget -O generate_copy.sh https://raw.githubusercontent.com/fossasia/yaydoc/master/generate.sh
```

Comment:
Why? **This is** the repository you're working on. Why are you downloading it again?